solenoid/settings/base.py

```diff
@@ -425,7 +425,12 @@ def make_list(value):
 
 CELERY_BROKER_URL = os.getenv('REDIS_URL',
                               default='redis://localhost:6379/0')
-CELERY_BROKER_TRANSPORT_OPTIONS = {"max_retries": 3, "interval_start": 0,
-                                   "interval_step": 0.2, "interval_max": 0.5}
 CELERY_RESULT_BACKEND = os.getenv('REDIS_URL',
                                   default='redis://localhost:6379')
+CELERY_BROKER_TRANSPORT_OPTIONS = {
+    "max_retries": 5,
+    "interval_start": 0,
+    "interval_step": 0.2,
+    "interval_max": 0.5,
+    "max_connections": 20,
```

Comment:
I'm not sure how we are determining all of these values. For now, if this works, I'd leave it. In the future I'd consider whether moving all of these to ENV to allow adjusting without changing the code has any value. In many ways, other than reformatting code layout, all of the changes in this PR are values that may need to change again to tune performance and thus might be better pulled out of the code itself.

Reply:
Yeah I completely agree. All of the config for this app needs to be refactored and I'll create a ticket for that now.